app/src/main/res/values/strings.xml

```diff
@@ -557,7 +557,7 @@
     <string name="error_file_creation">The file can not be created</string>
     <string name="error_path_creation">The destination folder can not be created</string>
     <string name="error_permission_denied">Permission denied by the system</string>
-    <string name="error_ssl_exception">Secure connection failed</string>
+    <string name="error_ssl_exception">Could not connect securely</string>
```

Comment:
I don't like this change

Reply:
Why? It is consistent with the other types of errors.

Reply:
personally "Secure connection failed" sounds better to me but I guess this is subjective.

Reply:
@cool-student Sounding better or not, you can't really tell from that whether it is a current secure connection that fails, or establishing it that fails. Give that a shake through translation, and now it is the wrong one, only in a different language.